src/app/server/CommissioningWindowManager.h

```diff
@@ -50,16 +50,16 @@ class CommissioningWindowManager : public SessionEstablishmentDelegate, public a
         return CHIP_NO_ERROR;
     }
 
-    static constexpr System::Clock::Seconds16 MaxCommissioningTimeout()
+    System::Clock::Seconds16 MaxCommissioningTimeout()
```

Comment:
```suggestion
    System::Clock::Seconds16 MaxCommissioningTimeout() const
```
Though this could probably stay as a static constexpr too, without any changes to any other code, since "foo.bar()" can invoke a static method in c++.

Reply:
I have reverted the change to be  `static constexpr`.